src/plugins/RepairMan.cpp

```diff
@@ -2,15 +2,57 @@
 //
 // Copyright (c) 2017-2020 Alexander Kurbatov
 
+#include "Historican.h"
+#include "Hub.h"
 #include "RepairMan.h"
 #include "core/API.h"
 #include "core/Helpers.h"
 
+namespace {
+Historican gHistory("repairman");
+
+}  // namespace
+
 void RepairMan::OnStep(Builder*) {
-    if (gAPI->observer().GetCurrentRace() != sc2::Race::Terran)
+    if (gHub->GetCurrentRace() != sc2::Race::Terran || m_damaged_buildings.empty())
```

Comment:
Ah I am wrong here, this plugin used to recreate destroyed buildings, too. So tis part I ok.

Reply:
Yes, I got this wrong too at first. I changed the plugin to be activated only for Terrans, but later I saw the recreate-destroyed-buildings logic and had to change it all back. Probably, the recreate-destroyed-buildings should be in Builder or something, as it will likely want to consider current needs rather than just auto-recreate, if it isn't a needed tech building. But since I don't have a plan for that yet, just keeping it as it is currently.